kuma/attachments/management/commands/configure_github_social.py

```diff
@@ -0,0 +1,71 @@
+import fileinput, os, sys, urllib2
+
+from django.core.management.base import BaseCommand, CommandError
+from django.contrib.sites.models import Site
+from allauth.socialaccount.models import SocialApp
+
+LOCALHOST = 'localhost:8000'
+MDN_LOCALHOST = 'mdn.localhost'
+
+OVERWRITE_PROMPT = 'There\'s already a SocialApp for GitHub, if you want to overwrite it type "yes":'
+GITHUB_INFO = '\n'.join([
+    'Visit https://github.com/settings/developers and click "New OAuth App"',
+    'Set "Homepage URL" to "http://mdn.localhost:8000/" and Authorization callback URL to ' +
+    '"http://mdn.localhost:8000/users/github/login/callback/" respectively'
+])
+ENV_INFO = 'Putting SITE_ID and DOMAIN into .env'
+HOSTS_INFO = '\n'.join([
+    'Make sure your hosts file contains these lines:'
+    '127.0.0.1 kubernetes.docker.internal localhost demos mdn.localhost beta.mdn.localhost wiki.mdn.localhost',
```

Comment:
Ditto about the `kubernetes.docker.internal` line. Also, `localhost` should already be there.